nbclient/client.py

```diff
@@ -820,13 +879,13 @@ def output(self, outs, msg, display_id, cell_index):
             # default output behaviour (e.g. OutputWidget)
             hook = self.output_hook_stack[parent_msg_id][-1]
             hook.output(outs, msg, display_id, cell_index)
-            return
+            return None
```

Comment:
Had to reread the PEP on this, but when returning elsewhere with a value it is recommended to have a `return None` explicitly where exiting early without an object. So good catch and I'll have to remember that in the future!

Reply:
explicit is better than implicit! :-)

Reply:
Yes, this is enforced buy the type checker that complains with `error: Return value expected` otherwise.